agent-docs/perp-orderbook-release-readiness.md

```
@@ -0,0 +1,584 @@
+| `UsualCollateral.fork.c.sol` | | Collateral independent |
+| `SpotAccount.fork.c.sol` | | Spot accounts independent |
+
+---
+
+## 5. Devnet Omnibus Structure
+
+### 5.1 File: `packages/tomls/src/omnibus/reya_devnet.toml`
+
+Module includes (referencing devnet-specific configs where needed):
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)